cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -48,17 +51,36 @@ func (c *CStorPoolController) syncHandler(key string, operation common.QueueOper
 		return nil
 	}
 	cStorPoolGot.Status.Phase = apis.CStorPoolPhase(status)
+
```

Comment:
Also, the previous error handling for this above line of code is missing with the latest changes.